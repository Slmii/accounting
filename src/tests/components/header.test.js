//react-test-renderer
import React from 'react';
import Header from '../../components/header';
import { shallow } from 'enzyme';

test('should render Header correctly', () =>{
    const wrapper = shallow(<Header />);
    // MACTHING SNAPSNOT (A COMPONENT)
    expect(wrapper).toMatchSnapshot();
});
