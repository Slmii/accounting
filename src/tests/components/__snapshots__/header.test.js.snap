// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<nav
  className="navbar sticky-top navbar-expand-md navbar-dark bg-dark"
>
  <Link
    className="navbar-brand"
    replace={false}
    style={
      Object {
        "color": "white",
      }
    }
    to="/"
  >
    Home
  </Link>
  <button
    aria-controls="collapsingNavbar"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler navbar-toggler-right"
    data-target="#collapsingNavbar"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="collapsingNavbar"
  >
    <ul
      className="navbar-nav ml-auto"
    >
      <li
        className="nav-item"
      >
        <NavLink
          activeClassName="is-active"
          ariaCurrent="true"
          className="nav-link"
          to="/add"
        >
          Create
        </NavLink>
      </li>
      <li
        className="nav-item"
      >
        <NavLink
          activeClassName="is-active"
          ariaCurrent="true"
          className="nav-link"
          to="/account"
        >
          My Account
        </NavLink>
      </li>
      <li
        className="nav-item"
      >
        <NavLink
          activeClassName="is-active"
          ariaCurrent="true"
          className="nav-link"
          to="/about"
        >
          About
        </NavLink>
      </li>
      <li
        className="nav-item"
      >
        <NavLink
          activeClassName="is-active"
          ariaCurrent="true"
          className="nav-link"
          to="/help"
        >
          Help
        </NavLink>
      </li>
    </ul>
  </div>
</nav>
`;
